CriticalMass/RulesDetailViewController.swift

```diff
@@ -7,19 +7,52 @@
 
 import UIKit
 
-class RuleDetailTextView: UITextView {
-    @objc
-    dynamic var ruleDetailTextColor: UIColor? {
-        willSet {
-            textColor = newValue
-        }
-    }
-}
-
 class RulesDetailViewController: UIViewController {
     private var rule: Rule
-    private var textView = RuleDetailTextView(frame: .zero)
-    
+    private let spacing: CGFloat = 12
+
+    private lazy var scrollView: UIScrollView = {
+        UIScrollView()
+    }()
+
+    private lazy var ruleStack: UIStackView = {
+        let view = UIStackView(arrangedSubviews: [ruleImageView, descriptionLabel])
+        view.axis = .vertical
+        view.distribution = .fill
+        view.spacing = 2 * spacing
+        return view
+    }()
+
+    private lazy var ruleImageView: UIImageView = {
+        let imageView = UIImageView(image: rule.artwork)
+        imageView.contentMode = .scaleAspectFit
+        return imageView
+    }()
+
+    private lazy var descriptionLabel: RuleDescriptionLabel = {
+        let label = RuleDescriptionLabel()
+        label.text = rule.text
+        return label
+    }()
+
+    private lazy var imageViewHeightConstraint = { () -> NSLayoutConstraint in
```

Comment:
Minor: how about `private lazy var imageViewHeightConstraint: NSLayoutConstraint = { ` to be consistent with the other lazy properties

Reply:
👍